.github/workflows/create-release-branch.yml

```diff
@@ -1,17 +1,6 @@
 name: create-release-branch
 on:
   workflow_dispatch:
-    inputs:
-      # When the workflow is executed manually, the user can select whether the branch should correspond to a major,
-      # minor, or patch release.
-      release-type:
-        type: choice
-        description: what kind of release?
-        options:
-          - major
-          - minor
-          - patch
-        required: true
   schedule:
```

Comment:
Note: Scheduled jobs only run in the default branch, so this won't run until we cycle the branches. We'll have to manually cut the v5 release branch every month by running the job manually.

Reply:
Since cron jobs run off the default branch, can we we instead have a cron job defined in dev that simple uses the github api to trigger (i.e. dispatch) the workflows. Basically using the github API... we theoretically should be able to do anything that you could do manually...

So maybe something like:
```
      - uses: actions/github-script@v6
        with:
          github-token: ${{ secrets.GH_TOKEN }}
          script: |
            await github.rest.actions.createWorkflowDispatch({
                owner: <owner>,
                repo: <repo>
                workflow_id: <workflow_id>,
                ref: <branch>,
            });
```

For example: See https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#create-a-workflow-dispatch-event and https://github.com/actions/github-script/issues/246

it looks like [the github-script readme](https://github.com/actions/github-script?tab=readme-ov-file) says `github` gives you a pre-authenticated [octokit/rest.js](https://octokit.github.io/rest.js) client with pagination plugins. Thus https://octokit.github.io/rest.js/v20#actions-create-workflow-dispatch is the documentaiton of github.reset.actions.createWorkflowDispatch